package.json

```diff
@@ -6,7 +6,10 @@
   "scripts": {
     "start": "gulp",
     "website": "gulp website",
-    "test": "gulp scripts"
+    "test": "npm run ts-test && npm run eslint",
+    "ts-test": "gulp scripts",
+    "eslint": "eslint --config ./.eslintrc.js --ext ts ./src/scripts",
+    "eslint-fix": "eslint --config ./.eslintrc.js --ext ts --fix ./src/scripts"
```

Comment:
Can you remove `test` and `ts-test` as their names are somewhat misleading, in travis we already run `tsc` so I don't think running `gulp-scripts` has any additional benefit

Reply:
I think it's probably worth keeping them for testing locally?

Reply:
I would say it would be better to use `gulp scripts` overall because the output is nicer,
But it doesn't throw on semantic errors right?

Reply:
What if we change `test` to something like `validate-all` so that is nice to set as a pre-commit hook.
```
    "validate-all": "gulp scripts && tsc && npm run eslint",
```

Reply:
`tsc` doesn't seem to be something that is automatically installed? doesn't work on my system.